@vates/parse-duration/parse-duration.test.mjs

```diff
@@ -0,0 +1,22 @@
+import { describe, it } from 'node:test'
+import assert from 'node:assert/strict'
+import { parseDuration } from '@vates/parse-duration/index.js'
+
+describe('`ms` without magic: always parse a duration and throws if invalid.', () => {
+  it('should parse string "2 days"', () => {
+    const input = '2 days'
+    const expected = 172800000
+    assert.strictEqual(parseDuration(input), expected)
+  })
+
+  it('should return the same duration if it is already in milliseconds', () => {
+    const input = 172800000
+    const expected = 172800000
+    assert.strictEqual(parseDuration(input), expected)
+  })
+
+  it('should throw an error for an undefined duration', () => {
+    const input = undefined
+    assert.throws(() => parseDuration(input), { message: 'not a valid duration: undefined' })
+  })
```

Comment:
I like to generate my tests automatically to be able to add more cases easily: https://github.com/vatesfr/xen-orchestra/blob/073b15ed0711261152c858898b7228135f683231/%40vates/otp/index.test.mjs#L74-L89

For instance, we should test at least both error conditions.

Reply:
In the code I linked, a new test is generated from each case, which makes the test report clearer :slightly_smiling_face: 

What's missing in the example I linked above is having different test title for different cases, either by specifying them manually (https://github.com/vatesfr/xen-orchestra/blob/530c30fd50c62a5b949b755768642b076f092d99/%40xen-orchestra/fs/src/path.test.js#L7-L16) or by generating them automatically from the values (https://github.com/vatesfr/xen-orchestra/blob/530c30fd50c62a5b949b755768642b076f092d99/%40vates/json-hash/index.test.js#L7-L21).